### PR TITLE
Embed words file and add go.mod file

### PR DESCRIPTION
**Pull request by adg**
A couple of modernizations.

### CODE REVIEW

File: typo.go [hunk 1]
```diff
@@ -17,40 +19,37 @@
 //
 // See the comments in the source for a description of the algorithm, extracted
 // from Bell Labs CSTR 18 by Robert Morris and Lorinda L. Cherry.
-//
```
Comment (robpike) [inline typo.go:20]: why did you delete this line?

File: typo.go [hunk 2]
```diff
@@ -4,7 +4,9 @@
 
 // Typo is a modern version of the original Unix typo command, a scan of whose man page
 // is at
-//   https://github.com/robpike/typo/blob/master/typo.png
+//
```
Comment (robpike) [inline typo.go:7]: why double this?

File: typo.go (hunk 1 above)
Comment (adg) [reply typo.go:20]: This is what gofmt does today.

File: typo.go (hunk 2 above)
Comment (adg) [reply typo.go:7]: ditto.

File: typo.go (hunk 1 above)
Comment (robpike) [reply typo.go:20]: Well, I wrote the proverb.

File: typo.go (hunk 2 above)
Comment (robpike) [reply typo.go:7]: really? one before and two after? well, if this is gofmt output, ok.

**Review by robpike — APPROVED** [review]

File: typo.go (hunk 2 above)
Comment (adg) [reply typo.go:7]: I don't make the rules. :)
